.goreleaser.yml

```diff
@@ -15,7 +15,7 @@ builds:
       post:
         - upx "{{ .Path }}"
     ldflags:
-      - -s -w -X 'github.com/achannarasappa/ticker/cmd.Version={{.Version}}'
+      - -s -w -X 'github.com/achannarasappa/ticker/v4/cmd.Version={{.Version}}'
```

Comment:
Pretty sure these should remain with a `/v4/` but not certain. Have you run a test build to confirm this should be here?